app/views/users/_form.html.erb

```diff
@@ -428,6 +428,10 @@
           <%= f.check_box :dl_locked_resources %>
           <span><strong>Download Restricted Resources</strong> - This person can access Restricted folders in the Staff Resources section.  Download Resources permission needed too.</span>
         </label>
+        <label class="checkbox">
+          <%= f.check_box :active %>
+          <span><strong>Active Volunteer</strong>- This person is an active volunteer. </span>
```

Comment:
Shouldn't change anything, but extra whitespace at the end. Maybe worth a more descriptive comment - what makes someone active, or what makes an inactive volunteer different from an active volunteer?